content/api/entity-reference/accounts-payable/index.html

```diff
@@ -9,992 +9,10 @@
-            <tr>
-            <tr>
-                <td>issue_1099s</td>
-                <td>boolean</td>
-                <td>Whether to issue 1099s to this vendor.</td>
-            <tr>
-            <tr>
-                <td>addresses</td>
-                <td>array of <a href="#VendorAddress">vendor addresses</a></td>
-                <td>The address(es) of the vendor.</td>
-            <tr>
-            <tr>
-                <td>vendor_payment_default</td>
-                <td><a href="#VendorPaymentDefault">vendor payment default</a></td>
-                <td>The default payment of this vendor.</td>
-            <tr>
-        </tbody>
-    </table>
-</div>
-
-<a id="VendorNoteAction"></a>
-<p>&nbsp;</p>
-<h2>Vendor note action</h2>
-<p>The <b>vendor note action</b> entity has the following properties:</p>
-<div class="table-responsive">
-    <table class="table table-striped table-hover">
-        <thead>
-            <tr>
-                <th><strong>Property</strong></th>
-                <th><strong>Type</strong></th>
-                <th><strong>Description</strong></th>
-            </tr>
-        </thead>
-        <tbody>
-            <tr>
-                <td>note_action_id</td>
-                <td>integer</td>
-                <td>The note/action identifier.</td>
-            <tr>
-            <tr>
-                <td>note_type</td>
-                <td>string</td>
-                <td>The type - note or action.</td>
-            <tr>
-            <tr>
-                <td>date</td>
-                <td>string</td>
-                <td>The date for this note/action.</td>
-            <tr>
-            <tr>
-                <td>content</td>
-                <td>string</td>
-                <td>The content of this note/action.</td>
-            <tr>
-            <tr>
-                <td>description</td>
-                <td>string</td>
-                <td>The description for this note/action.</td>
-            <tr>
-            <tr>
-                <td>type</td>
-                <td>string</td>
-                <td>The type of the note/action.</td>
-            <tr>
-            <tr>
-                <td>is_complete</td>
-                <td>boolean</td>
-                <td>Whether this note/action is complete.</td>
-            <tr>
-            <tr>
-                <td>assign_to</td>
-                <td>string</td>
-                <td>To whom this note/action is assigned.</td>
-            <tr>
-            <tr>
-                <td>completed_date</td>
-                <td>string</td>
-                <td>The date on which this note/action was completed.</td>
-            <tr>
-        </tbody>
-    </table>
+<div ng-app="entityReferenceApp">
+  <entity-reference
+    title="Accounts Payable"
+    get-started-url="{{ stache.config.guide_getting_started }}"
+    swagger-url="https://developer.sky.blackbaud.com/docs/services/57a342f1d7dcde1c28a749fe/export?DocumentFormat=Swagger&ApiName=Accounts%20Payable">
```

Comment:
Also, we probably want to leave these using the original so that teams can opt-in to the dynamic Entity Reference page once their swagger is ensured to be correct. I know Constituent API, Gift, Fundraising aren't ready to use the swagger yet for this (we still have to do some modifications first).